src/start.ts

```diff
@@ -59,6 +60,9 @@ export default (component: RootTramOneComponent, target: ElementOrSelector) => {
 	// setup key queue for new observable stores when resolving mounts
 	setupKeyQueue(TRAM_KEY_QUEUE);
 
+	// setup key queue for global observable stores when resolving mounts
+	setupKeyQueue(TRAM_GLOBAL_KEY_QUEUE);
```

Comment:
not a huge fan of setting up a whole new key queue here, but there really isn't a better way to associate hooks used in the render with their elements.